README.md

```diff
@@ -1082,6 +1082,66 @@ Given I add a ldap entry with dn "uid=c1aa97e9-fdff-4b81-b468-0cb41be3e550,dc=co
 
 ---
 
+### 📦 Database
+
+1. Setup db client with connection string
+
+```gherkin
+Given I setup database with connection string "<connectionString>"
+
+# Example:
+Given I setup database with connection string "postgres://user:password@localhost:5432/db"
+```
+
+2. Setup db client with full connection details
+
+```gherkin
+Given I setup database with driver "<driver>" host "<host>" port <port> user "<user>" password "<password>" database "<database>"
+# Example:
+Given I setup database with driver "postgres" host "localhost" port 5432 user "user" password "password" database "db"
+```
+
+3. Search db results on table with filter and columns
```

Comment:
Why not executing directly request ?
Like: When I execute sql request <sql_request> ?